build.vsh

```diff
@@ -33,8 +33,21 @@ if vls_git_hash.exit_code != 0 {
 }
 os.setenv('VLS_BUILD_COMMIT', vls_git_hash.output.trim_space(), true)
 
-use_libbacktrace_flag := if cc == 'msvc' { '' } else { '-d use_libbacktrace' }
-cmd := 'v -g -gc boehm $use_libbacktrace_flag -cc $cc cmd/vls -o $full_vls_exec_path'
+mut buffer := ['v', 'cmd/vls']
```

Comment:
it is better to have all option arguments before the source file/folder, I would have put `buffer << 'cmd/vls'` as a last line before `cmd := buffer.join(' ')`